pennylane/interfaces/batch/autograd.py

```diff
@@ -174,7 +174,7 @@ def grad_fn(dy):
 
                 # Generate and execute the required gradient tapes
                 if _n == max_diff:
-                    with qml.tape.Unwrap(*tapes):
+                    with qml.tape.Unwrap(*tapes, set_trainable=False):

```

Comment:
A bug I discovered. I think by now, almost all cases of `qml.tape.Unwrap()` require `set_trainable=False` to work!!

In a new PR, I will remove the `set_trainable` logic from `Unwrap`, since it seems to be not needed/not required.